template.ejs

```diff
@@ -3,6 +3,8 @@
   <head>
     <meta charset="UTF-8">
     <title><%= webpackConfig.template.title %></title>
+    <meta name="description" content="webpack is a module bundler. It packs CommonJs/AMD modules i. e. for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand.">
```

Comment:
This should basically become `<meta name="description" content="<%= webpackConfig.template.description %>">`. The setup should be able to pick it up from YAML headmatter if you include descriptions there.